python/lsst/sattle/sattlePy.py

```diff
@@ -213,8 +239,7 @@ def find_corners(self, sat_coords, length):
         line to those endpoints, and growing the perpendicular line
         to a specific length. We also add a buffer to the satellite endpoints
         to ensure the entire satellite path is included."""
-        # TODO: currently assumes one point at a time but we want array math
-        # TODO: Confirm that the math doesn't care about wrapping around 360/90
+        # TODO: Confirm we always make the smallest region
```

Comment:
Either do this, or make a followup ticket please

Reply:
Work has been done on a separate ticket, but I will move it over to this one.